File: Chapter_4/project_4_1.c
#include <stdio.h>

int main(void)
{

    int user_input, first, second;

    printf("Please enter a 2 digit number: ");
    scanf("%d", &user_input);

    first = user_input / 10;
    second = user_input % 10;

    printf("The reversal is: %d%d\n", second, first);
    return 0;
}

File: Chapter_4/project_4_1.py
# user_input = int(input("Please enter a 2 digit number: "))
user_input = input("Please enter a 2 digit number: ")

# first = user_input // 10
# second = user_input % 10
first, second = list(user_input)
print(f"The reversal is {second}{first}")
